src/components/RatingsReviews/RatingsReviews.jsx

```diff
@@ -13,23 +13,26 @@ class RatingsReviews extends React.Component {
     this.state = {
       reviews: [],
       sort: 'relevant',
-      showAddReviewModal: false
+      showAddReviewModal: false,
+      reviewsToDiplay: 2
     }
     this.handleSort = this.handleSort.bind(this);
     this.toggleReviewModal = this.toggleReviewModal.bind(this);
+    this.setNumReviewsToDisplay = this.setNumReviewsToDisplay.bind(this);
   }
 
   componentDidUpdate(prevProps) {
     if (this.props.product !== prevProps.product) {
       axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/reviews', {
         headers: {'Authorization': `${API_KEY}`},
         params: {
-          count: 2,
-          page: 1,
+          count: 50,
```

Comment:
thoughtful to store the reviews on the first get request... i like reducing the time complexity like that, great job!